details.html

```diff
@@ -0,0 +1,26 @@
+<%inherit file="base_app.htm" />
+
+<%block name="middle">
+<div id="section">
+    ## %for p in products:
+    <div class="images">
+        %for i in p.image_urls():
+        <img id="thumbnail" class="product_images" alt="i.filename" src="${i}">
+        %endfor
+    </div>
+    <div><img id="image_detail" class="product_images" alt="${p.name}" src="${p.image_url()}"></div>
+    <div class="prod_info">
+        <h1>${p.name}</h1>
+        </br>
+        <div id="price">$${p.price}</div>
```

Comment:
In your CSS class, you can use #price::before { content: "$";}. That way the $ is always used in price ID divs, and your don't need the first $ in `$${p.price}'. That way it doesnt get confused with the python insert code.

Reply:
Ah yeah I think that is a great idea. Thanks for catching it!